packages/react/src/components/select/tests/SelectReducers.spec.ts

```diff
@@ -69,6 +69,15 @@ describe('Select', () => {
 
                 expect(oldState).toEqual(oldStateBefore);
             });
+
+            it('does not add a duplicate ID in the state', () => {
+                const oldState: ISelectState[] = [selectInitialState];
+                const newState: ISelectState[] = selectCompositeReducer(oldState, addSelect(id));
+                const newNewState: ISelectState[] = selectCompositeReducer(newState, addSelect(id)); // trying to add the same id againß
+
+                expect(newState[1].id).toBe(id);
+                expect(newNewState.length).toBe(2); // oldState lenght is 1.
```

Comment:
I would put some expect after the first add just to make the behaviour a bit more clear

```suggestion
                const oldState: ISelectState[] = [selectInitialState];
                const newState: ISelectState[] = selectCompositeReducer(oldState, addSelect(id));

                expect(newState.length).toBe(2); // oldState length is 1.                
                expect(newState[1].id).toBe(id);
                                
                const newNewState: ISelectState[] = selectCompositeReducer(newState, addSelect(id)); // trying to add the same id again

                expect(newNewState.length).toBe(2); // length didn't change
```